CentOS 5.0 install error: expected specifier-qualifier-list before ‘__u32’

System: CentOS 5.0
psutil version: 5.3.1
error:
```
[root@localhost python-linux64]# ./bin/python ./bin/pip install psutil
Collecting psutil
  Downloading http://10.1.100.100:3141/root/dev/+f/1e1/66c5455b6fdb4/psutil-5.3.1.zip (460kB)
    100% |████████████████████████████████| 460kB 11.8MB/s 
Building wheels for collected packages: psutil
  Running setup.py bdist_wheel for psutil ... error
  Complete output from command /home/python-linux64/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-KMkFNZ/psutil/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/tmpGbIqfZpip-wheel- --python-tag cp27:
  In file included from /tmp/tmpMtb2JN.c:1:
  /usr/include/linux/ethtool.h:18: error: expected specifier-qualifier-list before ‘__u32’
  /usr/include/linux/ethtool.h:35: error: expected specifier-qualifier-list before ‘__u32’
  /usr/include/linux/ethtool.h:52: error: expected specifier-qualifier-list before ‘__u32’
  /usr/include/linux/ethtool.h:60: error: expected specifier-qualifier-list before ‘__u32’
  /usr/include/linux/ethtool.h:66: error: expected specifier-qualifier-list before ‘__u32’
  /usr/include/linux/ethtool.h:74: error: expected specifier-qualifier-list before ‘__u32’
  /usr/include/linux/ethtool.h:83: error: expected specifier-qualifier-list before ‘__u32’
  /usr/include/linux/ethtool.h:179: error: expected specifier-qualifier-list before ‘__u32’
  /usr/include/linux/ethtool.h:201: error: expected specifier-qualifier-list before ‘__u32’
  /usr/include/linux/ethtool.h:226: error: expected specifier-qualifier-list before ‘__u32’
  /usr/include/linux/ethtool.h:239: error: expected specifier-qualifier-list before ‘__u32’
  /usr/include/linux/ethtool.h:248: error: expected specifier-qualifier-list before ‘__u32’
  /usr/include/linux/ethtool.h:254: error: expected specifier-qualifier-list before ‘__u32’
  /tmp/tmpMtb2JN.c:1:27: warning: no newline at end of file
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-2.7
  creating build/lib.linux-x86_64-2.7/psutil
  copying psutil/__init__.py -> build/lib.linux-x86_64-2.7/psutil
  copying psutil/_psaix.py -> build/lib.linux-x86_64-2.7/psutil
  copying psutil/_common.py -> build/lib.linux-x86_64-2.7/psutil
  copying psutil/_psosx.py -> build/lib.linux-x86_64-2.7/psutil
  copying psutil/_pswindows.py -> build/lib.linux-x86_64-2.7/psutil
  copying psutil/_psbsd.py -> build/lib.linux-x86_64-2.7/psutil
  copying psutil/_compat.py -> build/lib.linux-x86_64-2.7/psutil
  copying psutil/_pslinux.py -> build/lib.linux-x86_64-2.7/psutil
  copying psutil/_pssunos.py -> build/lib.linux-x86_64-2.7/psutil
  copying psutil/_psposix.py -> build/lib.linux-x86_64-2.7/psutil
  creating build/lib.linux-x86_64-2.7/psutil/tests
  copying psutil/tests/__init__.py -> build/lib.linux-x86_64-2.7/psutil/tests
  copying psutil/tests/test_misc.py -> build/lib.linux-x86_64-2.7/psutil/tests
  copying psutil/tests/test_connections.py -> build/lib.linux-x86_64-2.7/psutil/tests
  copying psutil/tests/test_sunos.py -> build/lib.linux-x86_64-2.7/psutil/tests
  copying psutil/tests/test_process.py -> build/lib.linux-x86_64-2.7/psutil/tests
  copying psutil/tests/test_windows.py -> build/lib.linux-x86_64-2.7/psutil/tests
  copying psutil/tests/__main__.py -> build/lib.linux-x86_64-2.7/psutil/tests
  copying psutil/tests/test_posix.py -> build/lib.linux-x86_64-2.7/psutil/tests
  copying psutil/tests/test_unicode.py -> build/lib.linux-x86_64-2.7/psutil/tests
  copying psutil/tests/test_contracts.py -> build/lib.linux-x86_64-2.7/psutil/tests
  copying psutil/tests/runner.py -> build/lib.linux-x86_64-2.7/psutil/tests
  copying psutil/tests/test_system.py -> build/lib.linux-x86_64-2.7/psutil/tests
  copying psutil/tests/test_memory_leaks.py -> build/lib.linux-x86_64-2.7/psutil/tests
  copying psutil/tests/test_bsd.py -> build/lib.linux-x86_64-2.7/psutil/tests
  copying psutil/tests/test_linux.py -> build/lib.linux-x86_64-2.7/psutil/tests
  copying psutil/tests/test_osx.py -> build/lib.linux-x86_64-2.7/psutil/tests
  running build_ext
  building 'psutil._psutil_linux' extension
  creating build/temp.linux-x86_64-2.7
  creating build/temp.linux-x86_64-2.7/psutil
  gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -DPSUTIL_POSIX=1 -DPSUTIL_VERSION=531 -DPSUTIL_LINUX=1 -DPSUTIL_ETHTOOL_MISSING_TYPES=1 -I/home/python-linux64/include/python2.7 -c psutil/_psutil_common.c -o build/temp.linux-x86_64-2.7/psutil/_psutil_common.o
  psutil/_psutil_common.c:55:7: warning: no newline at end of file
  gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -DPSUTIL_POSIX=1 -DPSUTIL_VERSION=531 -DPSUTIL_LINUX=1 -DPSUTIL_ETHTOOL_MISSING_TYPES=1 -I/home/python-linux64/include/python2.7 -c psutil/_psutil_posix.c -o build/temp.linux-x86_64-2.7/psutil/_psutil_posix.o
  In file included from psutil/_psutil_posix.c:29:
  /usr/include/linux/if_packet.h:52: error: expected specifier-qualifier-list before ‘__u32’
  In file included from psutil/_psutil_posix.c:44:
  psutil/_psutil_common.h:14:7: warning: no newline at end of file
  error: command 'gcc' failed with exit status 1
```

--- a/psutil/_psutil_posix.c
+++ b/psutil/_psutil_posix.c
@@ -26,6 +26,7 @@
 
 #if defined(PSUTIL_LINUX)
     #include <netdb.h>
+    #include <linux/types.h>
     #include <linux/if_packet.h>
 #elif defined(PSUTIL_BSD) || defined(PSUTIL_OSX)
     #include <netdb.h>

